.ci/schedule-tests/src/combine-statistics.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs')
+const path = require('path')
+const AdmZip = require('adm-zip')
+
+const processTextFile = (fileContent, durations) => {
+  const lines = fileContent.split(/\n/)
+
+  for (const line of lines) {
+    const [suite, testFile, duration] = line.split(/\s+/)
+    const durationInSeconds = parseFloat(duration)
+
+    if (!isNaN(durationInSeconds)) {
+      const key = `${suite}:${testFile}`
+      durations[key] = [...(durations[key] || []), durationInSeconds]
+    }
+  }
+}
+
+const calculateMedian = (arr) => {
+  const sortedArr = arr.sort((a, b) => a - b)
+  const middle = Math.floor(sortedArr.length / 2)
+
+  return sortedArr.length % 2 === 0
+    ? (sortedArr[middle - 1] + sortedArr[middle]) / 2
+    : sortedArr[middle]
+}
+
+const combineStatistics = (directoryPath, outputFilePath) => {
```

Comment:
some kind of sort of result, either based on the runtime, or based on the name of spec files, would be useful to make the generated `runtime.txt` consumable by human too.

Reply:
I'm going to convert it to JSON as other reviewers found the TSV format confusing as well.

Reply:
What additional information would you be looking for in the runtime.json file?

Reply:
I think it's good at present, we can `jq` to manipulate on it anyways